standalone/src/models/shipping-method/shipping-rate/index.ts

```diff
@@ -1,5 +1,22 @@
-export * as ShippingRateDraft from './shipping-rate-draft';
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatModelBuilder,
+} from './builders';
 
-export { default as random } from './builder';
-export { default as presets } from './presets';
```

Comment:
Even though the presets are empty, we need to keep exporting them (as part of each model).
You can see [here](https://github.com/commercetools/test-data/blob/main/standalone/src/models/store/product-selection-setting/index.ts) an example.